src/LoginSignup/LoginModal.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react';
+import { registerEmail } from './RegistrationAPI';
+import { Formik } from 'formik';
+
+class LoginModal extends Component {
+	state = {
+		email: ''
+	};
+
+	createUser = () => {
+		return registerEmail(this.state.email);
+	};
+
+	render() {
+		return (
+			<div>
+				<main className="loginContainer">
+					<h1>Sign Up</h1>
+					<Formik
+						initialValues={{ email: '' }}
+						onSubmit={(values) => {
+							this.setState(() => ({
```

Comment:
no need binding here, formik take care of it

Reply:
Are you talking about the `this.setState()`?